### PR TITLE
conda-smithy 3.26.3

### PR DESCRIPTION
**Pull request by beckermr**
Merge only after success.

This pull request was auto-generated by [rever](https://regro.github.io/rever-docs/)

### CODE REVIEW

**Comment by h-vetinari** [comment]
Thanks a lot for the release! 🙏

I think we should add
```
    - tomli >=1.0.0  # [py<311]
```
as a run-dep?

**Comment by h-vetinari** [comment]
Or, if we keep it noarch, then without the selector...

**Comment by beckermr** [comment]
Yep. Noarch is really good here so we'll just install it even if it is not used.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ requirements:
     - scrypt
     - license-expression
     - libarchive
+    - tomli  # for python <3.11, can remove in the future
```
Comment (beckermr) [inline recipe/meta.yaml:48]:
```suggestion
    - tomli >=1.0.0  # for python <3.11, can remove in the future
```

**Pull request merged**